### DISCUSSION TITLE
Adding a new platform to the CI

### DISCUSSION THREAD

**Discussion by IgorTodorovskiIBM** [vim/vim#16207] [General]
Hi, I work on the IBM z/OS platform and Vim builds on the platform.

I am curious what it would take to add the z/OS platform to the CI. Are there any particular requirements for enabling CI on another platform? Is there an example PR?

**Comment by chrisbra** [comment]
Thanks. Any help is appreciated and having run the test suite on z/OS is definitely helpful. We used to have a s390 test with Travis CI, but currently we do not have it and rely on Debian to run the tests on that architecture. I started with a qemu/s390 CI build here: https://github.com/vim/vim/pull/12653 but I closed it again, since there were too many flaky tests. But perhaps you can pick it up from there.
